pyproject.toml

```diff
@@ -2,6 +2,26 @@
 requires = ["setuptools>=60", "setuptools_scm[toml]>=7.0"]
 build-backend = "setuptools.build_meta"
 
+[tool.coverage.run]
+parallel = true
+branch = true
+relative_files = true
+source = ["src/briefcase"]
+omit = [
+    "src/briefcase/__init__.py",
```

Comment:
Added this because it only contains version identification code, and it's impossible to branch check both branches; you either have setuptools_scm or you don't in the test environment; both are possible situations, so `#pragma` won't help here.